recipes/_test_vhost.rb

```diff
@@ -1,3 +1,6 @@
+# Expands size of server name hash buckets
+node.default['nginx']['server_names_hash_bucket_size'] = 128
```

Comment:
This doesn't exist in the version we're requiring, so I think it should just be removed.